.gitignore

```diff
@@ -16,3 +16,4 @@ selenium-debug.log
 *.iml
 
 **/.nyc_output
+package-lock.json
```

Comment:
https://docs.npmjs.com/files/package-lock.json
=> This file is intended to be committed into source repositories, and serves various purposes: